contracts/src/interfaces/IGenesis.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity ^0.8.23;
+
+/// @notice A interface that indicated the implementing facet contains a or multiple genesis settings.
+interface IGenesisComponent {
+    /// @notice Returns the id of the component
+    function id() external view returns(bytes4);
+
+    /// @notice Returns the actual bytes of the genesis
+    function genesis() external view returns(bytes memory);
+
+    /// @notice Checks if the component is bootstrapped
+    function bootstrapped() external view returns(bool);
```

Comment:
This is better placed in a Lifecycle facet and generalized as a `stage()` method that returns the lifecycle stage of a subnet.